src/main/java/org/broadinstitute/consent/http/models/Dataset.java

```diff
@@ -287,6 +289,14 @@ public void setSharingPlanDocument(String sharingPlanDocument) {
         this.sharingPlanDocument = sharingPlanDocument;
     }
 
+    public String getSharingPlanDocumentName() {
+        return sharingPlanDocumentName;
+    }
+
+    public void setSharingPlanDocumentName(String sharingPlanDocumentName) {
+        this.sharingPlanDocumentName = sharingPlanDocumentName;
+    }
```

Comment:
should this be added to the dataset schema in swagger?

Reply:
Yes, thank you!